p2p/peer.py

```diff
@@ -195,10 +198,21 @@ def __init__(self,
             self, cmd: protocol.Command, msg: protocol._DecodedMsgType) -> None:
         raise NotImplementedError("Must be implemented by subclasses")
 
-    def add_subscriber(self, subscriber: 'asyncio.Queue[PEER_MSG_TYPE]') -> None:
+    @property
+    def received_msgs_count(self) -> int:
+        return sum(self.received_msgs.values())
+
+    @property
+    def uptime(self) -> str:
+        delta = datetime.now() - self.start_time
+        hours, remainder = divmod(delta.seconds, 3600)
```

Comment:
I think you want `delta.total_seconds()` here.  Also, we should probably include days in this though being connected for more than 24 hours isn't likely in these early stages, it's easy to support now